src/libstore/globals.hh

```diff
@@ -142,6 +142,30 @@ public:
         )",
         {"build-cores"}, false};
 
+    Setting<unsigned int> loadLimit{
+      this,
+      getDefaultCores(),
+      "load-limit",
+      R"(
+        Sets the value of the `NIX_LOAD_LIMIT` environment variable in the
+        invocation of builders. Builders can use this value at their discretion
+        to dynamically control the amount of parallelism with respect to the
+        machine's load average. For instance, in Nixpkgs, if this value is >0,
+        the builder passes the `-lN` flag to GNU Make. In this case, if the
+        load average of the machine exceeds `N`, the amount of parallelism will
+        be dynamically reduced to 1.
```

Comment:
```suggestion
        machine's load average.
        
        For instance, a builder could use the value to set the `-l` flag to GNU Make.
        In this case, if the load average of the machine exceeds `NIX_LOAD_LIMIT`, the amount of parallelism will be dynamically reduced to 1.
```

No need to point to a specific implementation, as we'd have to keep track of if that statement is still correct.